go/conversation/tests.py

```diff
@@ -353,6 +356,10 @@ def test_post_new_conversation_starting(self):
 
 
 class TestConversationViews(BaseConversationViewTestCase):
+
+    with_event_statuses = mock.patch(
+        'go.conversation.settings.ENABLE_EVENT_STATUSES_IN_MESSAGE_LIST', True)
+
```

Comment:
Can we perhaps use `self.monkey_patch()` in the tests that need it instead of `mock`? Using different patching mechanisms in different places is confusing.